todo_app/SeleniumTest/Tests/test_e2e.py

```diff
@@ -0,0 +1,41 @@
+from os import environ, path
+from threading import Thread
+#from todo_app.data.trello import createNewBoard, deleteBoard
+from todo_app import app
+import pytest
+from unittest.mock import patch, Mock
+from selenium import webdriver
+import os
+from todo_app.data.trello  import Trello
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    trello =Trello()
+    # Create the new board & update the board idenvironment variable
+    board_id = trello.createNewBoard('SeleniumNewBoard')
+    os.environ['TRELLO_BOARD_ID'] = board_id
```

Comment:
You override the board id, but not the list ids which means any more complex interactions will fail. We should add a call to override those as well, e.g. we could use a function something like:
```
def find_and_set_todo_list_id():
    trello = Trello()
    ### TODO Add this function to the Trello class
    todo_list = trello.get_list("To Do")
    os.environ['ID_LIST_TODO'] = todo_list['id']
```